### ISSUE TITLE
ListView has been removed from React Native.

### ISSUE THREAD

**Issue by dj1993cse** [bietkul/react-native-form-builder#42]
package.json:->

{
  "name": "FormBuilderApp",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "react-native start",
    "test": "jest",
    "lint": "eslint ."
  },
  "dependencies": {
    "native-base": "^2.13.7",
    "react": "16.8.6",
    "react-native": "0.60.5",
    "react-native-form-builder": "^1.0.16"
  },
  "devDependencies": {
    "@babel/core": "^7.5.5",
    "@babel/runtime": "^7.5.5",
    "@react-native-community/eslint-config": "^0.0.5",
    "babel-jest": "^24.9.0",
    "eslint": "^6.3.0",
    "jest": "^24.9.0",
    "metro-react-native-babel-preset": "^0.56.0",
    "react-test-renderer": "16.8.6"
  },
  "jest": {
    "preset": "react-native"
  }
}

Error ScreenShot:->
![Capture4](https://user-images.githubusercontent.com/40656868/64341986-2d9b1f00-d007-11e9-9592-04e765be6ab1.JPG)

**Comment by amrs-tech** [comment]
Yeah, I too face this same issue. Is there anyway to override this to run the app normally ?
Or else is there any fix for this issue to get solved, because I think it is due to the KeyboardAwareScrollView used as dependencies here. Please do provide a solution for this issue.
